python/src/aiconfig/editor/server/server.py

```diff
@@ -269,7 +270,18 @@ def delete_parameter() -> FlaskResponse:
     prompt_name: str | None = request_json.get("prompt_name")
 
     operation = make_op_run_method(MethodName("delete_parameter"))
-    operation_args: Result[OpArgs, str] = result.Ok(
-        OpArgs({"parameter_name": parameter_name, "prompt_name": prompt_name})
-    )
+    operation_args: Result[OpArgs, str] = result.Ok(OpArgs({"parameter_name": parameter_name, "prompt_name": prompt_name}))
     return run_aiconfig_operation_with_op_args(aiconfig, "delete_parameter", operation, operation_args)
```

Comment:
Is this auto-linting? Can you try to do that atomically for large sets of files in their own PRs, or even better do it before submitting the PR that modifies that specific code?

Reply:
You're right, my bad I forgot to do it last time